src/shadertools/modules/picking/picking.js

```diff
@@ -1,17 +1,25 @@
-const DEFAULT_HIGHLIGHT_COLOR = new Uint8Array([0, 64, 128, 64]);
+import {log} from '../../../utils';
```

Comment:
This is a copy paste of this file from deck.gl, which has been used for picking/highlighting for few months on master.